src/main/java/mods/railcraft/common/carts/MinecartHooks.java

```diff
@@ -206,18 +206,17 @@ public void onEntityCollision(EntityMinecart cart, Entity other) {
     public AxisAlignedBB getCollisionBox(EntityMinecart cart, Entity other) {
         if (other instanceof EntityItem && RailcraftConfig.doCartsCollideWithItems())
             return other.getEntityBoundingBox();
-        if (other instanceof EntityPlayer)
-            return other.canBePushed() ? other.getEntityBoundingBox() : null; //            return other.boundingBox.contract(COLLISION_EXPANSION, 0, COLLISION_EXPANSION);
-        return null;
+        return other.canBePushed() ? other.getEntityBoundingBox() : null; //            return other.boundingBox.contract(COLLISION_EXPANSION, 0, COLLISION_EXPANSION);
     }
 
     @Override
     public AxisAlignedBB getMinecartCollisionBox(EntityMinecart cart) {
-        double yaw = Math.toRadians(cart.rotationYaw);
-        double diff = ((CART_LENGTH - CART_WIDTH) / 2.0) + MinecartHooks.COLLISION_EXPANSION;
-        double x = diff * Math.abs(Math.cos(yaw));
-        double z = diff * Math.abs(Math.sin(yaw));
-        return cart.getEntityBoundingBox().expand(x, MinecartHooks.COLLISION_EXPANSION, z);
+        return cart.getEntityBoundingBox().expand(MinecartHooks.COLLISION_EXPANSION, 0, MinecartHooks.COLLISION_EXPANSION);
```

Comment:
Does this break the Tunnel Bore collision? Hmm....not sure why I do this.

Reply:
I think I added it when I was trying to fix the locomotive hitting people thing.